sleap_nn/train.py

```
@@ -0,0 +1,134 @@
+        peak_threshold=0.2,
+        make_labels=True,
+        save_path=Path(cfg.trainer_config.save_ckpt_path) / "pred_val.slp",
+    )
+
+    evaluator = Evaluator(
+        ground_truth_instances=val_labels, predicted_instances=pred_val_labels
+    )
+    dist_metrics = evaluator.distance_metrics()
+    try:
+        map_oks = evaluator.voc_metrics(match_score_by="oks")["oks_voc.mAP"]
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)